### PR TITLE
[Author] [New] Add country-region-dropdown-menu w/ git auto-update

### PR DESCRIPTION
**Pull request by geodatasource**
Pull request for issue: #10825 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/geodatasource/country-region-dropdown-menu
 * Official website (optional, not the repository): http://www.geodatasource.com/
 * NPM package url (optional): https://www.npmjs.com/package/country-region-dropdown-menu
 * License and its reference: https://github.com/geodatasource/country-region-dropdown-menu/blob/master/license.txt
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 1
   - Count of stars: 3
   - Count of forks: 0
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 31
   - Downloads in the last month: 31

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by dalisoft** [comment]
- Status
  - [ ] Review required: Waiting... (please wait up to 24 hours)
  - [x] Tests pass: OK
  - [ ] Waiting for popuality: Required at least 500 downloads on npm

File: ajax/libs/country-region-dropdown-menu/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "country-region-dropdown-menu",
+  "filename": "geodatasource-cr.min.js",
+  "version": "1.0.1",
+  "description": "A dropdown menu created with javascript that allows user to implement country-region dropdown list effortlessly.",
+  "homepage": "http://www.geodatasource.com/",
```
Comment (sufuf3) [inline ajax/libs/country-region-dropdown-menu/package.json]: Use `"homepage": "https://www.geodatasource.com/"` is better.

File: ajax/libs/country-region-dropdown-menu/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          "**/*"
+        ]
+      }
+    ]
+  },
+  "license": "LGPL-3.0",
+  "author": {
+    "name": "GeoDataSource",
+    "email": "support@geodatasource.com",
+    "url": "http://www.geodatasource.com/"
```
Comment (sufuf3) [inline ajax/libs/country-region-dropdown-menu/package.json]: Use `"url": "https://www.geodatasource.com/"` is better.

**Comment by geodatasource** [comment]
May I know what to fix in package.json to get v1.0.0 automatically?

**Comment by PeterDaveHello** [comment]
I'll squash and rebase this PR first, @cdnjs/intern2 please help review it.

**Comment by geodatasource** [comment]
@PeterDaveHello Thank you. Can you please assist on what to fix in package.json to get v1.0.0 automatically too?

**Comment by PeterDaveHello** [comment]
@geodatasource Let's see the CI testing result after the updates first :)

**Comment by geodatasource** [comment]
@PeterDaveHello sure. Is it a must to clone everything from geodatasource/cdnjs for the rebase of PR?

**Comment by maruilian11** [comment]
@geodatasource I think this PR is good.
 Could we also add a commit behind with yours to add `1.0.0` manually because the current auto-update filemap can't fetch `1.0.0`.  Thanks!

**Comment by PeterDaveHello** [comment]
@maruilian11 don't do that, that's what you should do.

**Comment by PeterDaveHello** [comment]
@maruilian11 so please help add it and write correct commit message.

**Comment by geodatasource** [comment]
@PeterDaveHello Any update on this?

**Comment by PeterDaveHello** [comment]
Ping @maruilian11

**Comment by maruilian11** [comment]
@PeterDaveHello hmmm... I said `we` means cdnjs team.
I will help add it.

**Comment by maruilian11** [comment]
Helped add `1.0.1` and improve the commit msg.

**Comment by PeterDaveHello** [comment]
@pvnr0082t @maruilian11 I think we'll only manually do that for libraries reached or close to the popularity requirement, for those not, the versions can't be fetched can be dropped, sorry for didn't notice earlier.

cc @cdnjs/intern @cdnjs/intern2 @cdnjs/intern3

**Comment by maruilian11** [comment]
got it.

**Pull request merged**